### DISCUSSION TITLE
MultiTargetTrack for MFADataAssociator

### DISCUSSION THREAD

**Discussion by jgbos** [dstl/Stone-Soup#860] [Q&A]
Is there an example of creating a multiple target tracker for multi-frame association?  I see the example example that doesn't utilize track initiation.  I tried to make my own but keep running into issues.  I can give more info but wanted to first ask if there was already documentation that I didn't find for this.

**Comment by sdhiscocks** [comment]
So we currently don't have an initiator for states for MFA, but #873 is adding the GaussianMixtureInitiator which will make MFA easier to use. Is this the main issue you were coming up against?
